spec/javascripts/models/beta/history_spec.js

```diff
@@ -0,0 +1,47 @@
+import * as History from 'models/beta/history';
+
+describe('History', () => {
+  describe("getDate", () => {
+    describe('when key is updated_at', () => {
+      describe('and is a valid date', () => {
+        const key = "updated_at";
+        const validDate = "2019/08/27 14:14:24 -0300";
+        const changes = {
+          updated_at: [validDate, validDate]
+        };
+
+        it("return a formated date", () => {
+          const [ formatedDate ] = History.getDate(key, changes);
+
+          expect(formatedDate).toEqual(I18n.l("date.formats.long", changes.updated_at[0]));
+        });
+      });
+
+      describe('and is a invalid date', () => {
+        const key = "updated_at";
+        const invalidDate = null;
+        const changes = {
+          updated_at: [invalidDate]
+        };
+
+        it("return null", () => {
+          const [ nullDate ] = History.getDate(key, changes);
+
+          expect(nullDate).toEqual(null);
+        });
+      });
+    });
+
+    describe('when key is state', () => {
+      const key = "state";
+      const changes = {
+        state: ["finished", "delivered"]
+      };
+
+      it("return just the changes", () => {
+        expect(History.getDate(key, changes)[0]).toEqual("finished");
```

Comment:
What do you think about doing something like that:

```suggestion
        expect(History.getDate(key, changes)).toEqual(["finished", "delivered"]);
```